releases/release-1.7/README.md

```diff
@@ -0,0 +1,56 @@
+# Kubeflow 1.7
+
+### Links
+
+- [Release Team](release-team.md)
+- [Calendar](https://arrik.to/kf-release-team-cal)
+- [Meeting Notes](https://arrik.to/kf-release-team-notes)
+- [Recordings](https://arrik.to/kf-release-team-recordings)
+- [Retrospective Document](https://bit.ly/kf-release-1-6-retro)
+- Project Board
+- Contact
+  - [#release](https://app.slack.com/client/T7QLHSH6U/C9V2WT2KV) on slack
+  - [@release-team](https://github.com/orgs/kubeflow/teams/release-team) on GitHub
+
+### TL;DR
+
+The 1.7 release cycle is proposed as follows
+
+- **Wednesday, Sep 28th 2022**: Week 1 - Release Team is formed
+- **Wednesday, Sep 28th 2022**: Week 1 - Release Cycle Begins
+- **Wednesday, Oct 12th 2022**: Week 3 - Roadmaps are finalized
+- **Wednesday, Jan 18th 2023**: Week 17 - Feature Freeze
+- **Wednesday, Jan 25th 2023**: Week 18 - Manifests Testing Starts
+- **Wednesday, Feb 1st 2023**: Week 19 - Manifests Testing Ends
+- **Wednesday, Feb 1st 2023**: Week 19 - Distribution Testing Starts
+- **Wednesday, Feb 22nd 2023**: Week 22 - Distribution Testing Ends
+- **Tuesday, Feb 28th 2023**: Week 23 - Docs Update Ends
```

Comment:
tagging previous distribution owners to give feedback

@julioo @VaishnaviHire @LaVLaS @johnugeorge @yhwang @zijianjoy @gkcalat @surajkota @kimwnasptd

Reply:
looks good to me

Reply:
LGTM
One question: it seems that we are planning to update docs before releasing Kubeflow. Could they happen on the same day?

Reply:
@gkcalat there might be a confusion between Docs Updates and Docs Publishing. We expect all the docs changes that working groups or distribution owners have to be ready the day before so that we don't have to wait for any late changes on the day of the release and just publish the documentation. Does that make sense?

Reply:
@DomFleischmann thank you. Sounds good. The issue I mentioned is that the website is watching the master branch. As long as we merge new docs in the repo, the updated docs go live.